__tests__/app.test.js

```diff
@@ -45,3 +46,10 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+// describe('GET /api/articles/article_id', () => {
```

Comment:
Technically this should be on an independent branch to this one, so it doesn't muddy the code. 

I'd just make sure you're branching from main, not the previous branch, as it causes issues if you have to go back and change something